simulation/main.go

```diff
@@ -0,0 +1,333 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/ark-network/ark/pkg/client-sdk/store"
+	"github.com/ark-network/ark/pkg/client-sdk/types"
+	"os"
+	"sync"
+	"time"
+
+	arksdk "github.com/ark-network/ark/pkg/client-sdk"
+	utils "github.com/ark-network/ark/server/test/e2e"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/xeipuuv/gojsonschema"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	composePath = "../docker-compose.clark.regtest.yml"
+)
+
+var (
+	aspUrl     = "localhost:7070"
+	clientType = arksdk.GrpcClient
+	password   = "password"
+	walletType = arksdk.SingleKeyWallet
+
+	tempDirs      []string
+	tempDirsMutex sync.Mutex
+)
+
+func main() {
+	simFile := flag.String("simulation", "simulation1.yaml", "Path to the simulation YAML file")
+	flag.Parse()
+
+	simulation, err := loadAndValidateSimulation(*simFile)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Infof("Simulation Version: %s\n", simulation.Version)
+	log.Infof("ASP Network: %s\n", simulation.Server.Network)
+	log.Infof("Number of Clients: %d\n", len(simulation.Clients))
+	log.Infof("Number of Rounds: %d\n", len(simulation.Rounds))
+
+	roundLifetime := fmt.Sprintf("%d", simulation.Server.RoundInterval)
```

Comment:
It seems to me that the content of the docker-env temp file would look like the following with a round interval of let's say 10 seconds:

```
10
```

instead this is what we want?

```
ARK_ROUND_INTERVAL=10
```

```suggestion
	roundInterval := fmt.Sprintf("ARK_ROUND_INTERVAL=%d", simulation.Server.RoundInterval)
```

Reply:
Done